packages/storage-driver-cloudinary/src/index.ts

```diff
@@ -79,6 +79,10 @@ export class DriverCloudinary implements Driver {
 		return String(new Date().getTime());
 	}
 
+	private getLatestVersion() {
+		return `v${this.getTimestamp()}`;
```

Comment:
Would be nice if this is a known last updated timestamp rather than _now_ to make it a bit more efficient)